### PR TITLE
--active flag to filter the active blobbers

### PR DESCRIPTION
**Pull request by chauchausoup**

### CODE REVIEW

File: network/one.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
-block_worker: http://one.devnet-0chain.net/dns
```
Comment (andrenerd) [inline network/one.yaml:2]: please, revert it.
the file is called "one.yaml". it's supposed, i guess, the one-network settings should be there. 
for six-network settings could be put in six.yaml. and probably six.yaml should be on ignored list, not keep it locally only.

File: cmd/storage.go [hunk 2]
```diff
@@ -91,15 +109,27 @@ var lsBlobers = &cobra.Command{
...
+				util.PrintJSON(filterActiveBlobbers(list))
+				return
+			}
+			printBlobbers(filterActiveBlobbers(list))
+		} else {
+			if doJSON {
+				util.PrintJSON(list)
+				return
+			}
+			printBlobbers(list)
```
Comment (andrenerd) [inline cmd/storage.go:131]: sorry, for being too picky.
could it be done on "data" layer, not "presentation" layer.  not spitting `print` function between scenarios. also lets drop excessive code.

```
if doActive {
    list = filterActiveBlobbers(list);
} 

if doJSON {
    util.PrintJSON(list)
} else {
    printBlobbers(list)
}
```

File: cmd/storage.go [hunk 3]
```diff
@@ -83,6 +84,23 @@ func printBlobbers(nodes []*sdk.Blobber) {
...
+
+		} else {
+			fmt.Println("Inactive Blobber: ", blobbers[i].BaseURL)
+		}
+
+	}
+
+	return result
+}
+
```
Comment (andrenerd) [inline cmd/storage.go:103]: let's drop excessive code. at first is there need to multiply by1000 the both sides of the comparison ;)? also lets apply existing utility methods for the Timestamp struct. and finally, named return param would be really helpful in this very function. so get:

```
func filterActiveBlobbers(blobbers []*sdk.Blobber) (activeBlobbers []*sdk.Blobber) {
    for i := range blobbers {
        if blobbers[i].LastHealthCheck.Within(60 * 60) {
            activeBlobbers = append(activeBlobbers, blobbers[i])
        }
    }
   return
}
```

PS. ideally such function should be added, as a method of the blobber list struct, right to the SDK.

**Review by andrenerd — CHANGES_REQUESTED** [review]

File: cmd/storage.go (hunk 3 above)
Comment (chauchausoup) [reply cmd/storage.go:103]: Yes this funciton must be added  in SDK layer.

Comment (chauchausoup) [reply cmd/storage.go:103]: What should we do then, make updates in SDK and then changes in zboxcli for merge?
Or for now implement funciton in cli and after making updates in SDK again another merge?
@andrenerd

Comment (andrenerd) [reply cmd/storage.go:103]: Checked the sdk codebase, there is no "[]*Blobber" struct there.
Well, in such case, lets add the function right here (zbocli), not extending sdk for now.

**Comment by chauchausoup** [comment]
Made required changes . Please check.
@andrenerd

**Review by andrenerd — APPROVED** [review]

**Comment by guruhubb** [comment]
@chauchausoup see https://github.com/0chain/blobber/issues/208

**Review by andrenerd — APPROVED** [review]

**Pull request merged**